LinkedList.c

```diff
@@ -0,0 +1,181 @@
+#include<stdio.h>
+#include<stdlib.h>
+struct NODE {
+	int d;		//integer type data  
+	struct NODE* next;		//pointer to same structure
+}*start;	//first node of the list
+
+int main() {
+	start=NULL;
+	
+	// Adding element to the list
+	addFront(23);
+    traverse();
+             
+	// Adding elements at the end
+    addLast(25);
+    addLast(46);
+    traverse();
+         
+    // Adding elements to front
+    addFront(12);
+    addFront(11);
+	traverse();

```

Comment:
That's because your file contains mixed spaces and tabs. Try selecting the blank space with your mouse and you will know. As this code is C, please switch it to use all tabs.